addons/gamification/tests/common.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo, TransactionCaseWithUserDemo
+
+
+class HttpCaseWithUserDemoGamification(HttpCaseWithUserDemo):
+    def setUp(self):
+        super().setUp()
+        if not self.user_demo.karma:
+            self.user_demo.karma = 2500
+
+
+class TransactionCaseWithUserDemoGamification(TransactionCaseWithUserDemo):
+    def setUp(self):
```

Comment:
I'll add a reminder for the FW in 16.0 => The setUp was replaced by setUpClass for TransactionCaseWithUserDemo